crates/bevy_gizmos/src/aabb.rs

```diff
@@ -16,7 +16,7 @@ use bevy_reflect::{std_traits::ReflectDefault, Reflect};
 use bevy_render::primitives::Aabb;
 use bevy_transform::{
     components::{GlobalTransform, Transform},
-    TransformSystem,
+    plugins::TransformSystem,
```

Comment:
I feel like these should be re-exported top-level from `bevy_transform`. For all of this stuff, there's no reason to touch these files, and it increases breakage for people who may already use transform internals.

Reply:
Resolved in https://github.com/bevyengine/bevy/pull/13599